rmf_demo_panel/README.md

```diff
@@ -0,0 +1,52 @@
+# RMF Demo Panel
+
+This will be a replacement of rviz's [rmf_panel](https://github.com/osrf/rmf_schedule_visualizer)
+
+## Installation
+Dependencies
+ - rmf_core: `feature/task-dispatcher` branch
```

Comment:
We just merged in `feature/task-dispatcher` into `develop/dispatcher_demo`. Would appreciate if you can also update this. Thanks

Reply:
right on it.

just wanted to check w you, when you mean `Time is XXX` does not show up, it's only the numerals not showing up or the entire line including `Time is`??